presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorTableStatisticsProvider.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.spi.connector;
+
+import com.facebook.presto.spi.ConnectorSession;
+import com.facebook.presto.spi.ConnectorTableLayoutHandle;
+import com.facebook.presto.spi.statistics.TableStatistics;
+
+public interface ConnectorTableStatisticsProvider
+{
+    TableStatistics getTableStatistics(ConnectorTransactionHandle transactionHandle, ConnectorSession session, ConnectorTableLayoutHandle layoutHandle);
```

Comment:
We only want to do name mapping once. Also consider things like mixed case or quoted identifiers. At some point, we're going to need to handle that properly instead of just lower asking everything, so we shouldn't do name matching in multiple places. Everything else in the SPI uses ColumnHandle.